compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/FirErrorsDefaultMessages.kt

```diff
@@ -1051,43 +1052,44 @@ object FirErrorsDefaultMessages : BaseDiagnosticRendererFactory() {
             RENDER_TYPE
         )
 
-        // Inline and value classes
-        map.put(INLINE_CLASS_NOT_TOP_LEVEL, "Inline classes cannot be local or inner")
-        map.put(INLINE_CLASS_NOT_FINAL, "Inline classes can be only final")
-        map.put(ABSENCE_OF_PRIMARY_CONSTRUCTOR_FOR_INLINE_CLASS, "Primary constructor is required for inline class")
-        map.put(INLINE_CLASS_CONSTRUCTOR_WRONG_PARAMETERS_SIZE, "Inline class must have exactly one primary constructor parameter")
+        // Value classes
+        map.put(VALUE_CLASS_NOT_TOP_LEVEL, "Value classes cannot be local or inner")
+        map.put(VALUE_CLASS_NOT_FINAL, "Value classes can be only final")
+        map.put(ABSENCE_OF_PRIMARY_CONSTRUCTOR_FOR_VALUE_CLASS, "Primary constructor is required for value class")
+        map.put(VALUE_CLASS_CONSTRUCTOR_WRONG_PARAMETERS_SIZE, "Inline class must have exactly one primary constructor parameter") // +
```

Comment:
Meaningless comment

Reply:
According to comment in the beginning of the file this means that a better message is expected. In this case I set it as value classes may have more than 1 argument but it is not yet supported.